.github/workflows/tests.yml

```diff
@@ -126,16 +199,37 @@ jobs:
           docker build --no-cache -t icecube/skymap_scanner .
       - name: Run
         run: |
-          jq '.GCDQp_packet' tests/data/reco_pixel_pkls/${{ matrix.dir }}/startup.json > tests/data/reco_pixel_pkls/${{ matrix.dir }}//GCDQp_packet.json
+          # grab the GCDQp_packet key and throw into a file
+          jq '.GCDQp_packet' \
+            tests/data/reco_pixel_pkls/${{ matrix.dir }}/startup.json > \
+            tests/data/reco_pixel_pkls/${{ matrix.dir }}//GCDQp_packet.json
+
+          # move python file to subdir to make bindings simpler
+          cp tests/data/reco_pixel_pkls/get_toclient_msg_pkl.py \
+            tests/data/reco_pixel_pkls/${{ matrix.dir }}
+
+          # create the in.pkl
+          docker run --network="host" --rm -i \
+            --shm-size=6gb \
+            --mount type=bind,source=$(readlink -f tests/data/reco_pixel_pkls/${{ matrix.dir }}),target=/local/pkls \
+            --env PY_COLORS=1 \
+            $(env | grep '^SKYSCAN_' | awk '$0="--env "$0') \
+            icecube/skymap_scanner:latest \
```

Comment:
It seems wrong that we'd use a docker image from `main`.  What if the PR is editing something that affects this?

Is there actually a reason we need to use docker?  Would a local python install or venv not suffice?

Reply:
It needs `icecube`/`icetray`

Reply:
In contrast, `tests/compare_scan_results.py` doesn't need `icecube` (see several lines above)

Reply:
> It seems wrong that we'd use a docker image from main. What if the PR is editing something that affects this?

Can you elaborate on this?

Reply:
Ah, one thing I think I didn't catch. You're running 
```
docker build --no-cache -t icecube/skymap_scanner .
```
in a previous step, which locally builds `icecube/skymap_scanner:latest`.

I'd recommend using a different tag, to disambiguate from the global `icecube/skymap_scanner:latest` that gets built for the `main`/`master` branch and pushed to dockerhub.  Because you technically can't guarantee with this usage that it's not pulling from dockerhub.

Reply:
Fair point, I suppose that's good practice in general not to tag a local image as `latest`